### PR TITLE
layer.md: Fix some typo

### PR DESCRIPTION
**Pull request by TrumanLing**
Signed-off-by: Ling FaKe <lingfake@huawei.com>

### CODE REVIEW

File: layer.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ In the absence of any [whiteout](#whiteouts) files in a layer changeset, the arc
 This section covers applying an entry in a layer changeset, if the file path already exists.
 
 If the file path is a directory, then the existing path just has it's attribute set from the layer changeset for that filepath.
-If the file path is any other file type (regular file, FIFO, etc), then the:
+If the file path is any other file type (regular file, FIFO, etc), then:
 * file path is unlinked (See [`unlink(2)`](http://linux.die.net/man/2/unlink))
```
Comment (wking) [inline layer.md]: +1 to removing “the”, but we should also rephrase this to an imperative:

> unlink the file path (…)

But this section is getting overhauled in #317, so it may not be worth the trouble of fixing it here ;).

Comment (TrumanLing) [inline layer.md]: @wking I have removed modification you mentioned. PTAL.

**Comment by wking** [comment]
ff1291c1 looks good to me, but it would be easier to identify this
change in the logs if the commit message were something like:

  layer: Fix "an copy" -> "a copy" typo

**Comment by TrumanLing** [comment]
@wking thanks for you suggestion. It is OK. PTAL

**Comment by wking** [comment]
Looks great, thanks :)

**Comment by vbatts** [comment]
LGTM

**Comment by philips** [comment]
LGTM
